wgpu-core/src/instance.rs

```diff
@@ -896,6 +896,18 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             // hardware GPU (integrated or discrete).
             PowerPreference::LowPower => integrated.or(discrete).or(other).or(virt).or(cpu),
             PowerPreference::HighPerformance => discrete.or(integrated).or(other).or(virt).or(cpu),
+            PowerPreference::None => {
+                let option_min = |a: Option<usize>, b: Option<usize>| {
+                    if let (Some(a), Some(b)) = (a, b) {
+                        Some(a.min(b))
+                    } else {
+                        a.or(b)
+                    }
+                };
+                // Pick the lowest id of these types (which will give us a `Backends::PRIMARY`
```

Comment:
This comment isn't right - it just gives us a non-cpu or virtualized adapter

Reply:
The lowest id will be a `Backends::PRIMARY`, right (if any of them are)?

Reply:
Bump - the non-cpu or virtualized adapter will be Backends::PRIMARY if possible, right?

Reply:
Not necessarily - it will give them to us in backend preference order, but the actual situation we're dealing with is where there is a primary backend CPU adapter and a secondary backend GPU adapter. We want to make sure we choose the GPU adapter.

Reply:
I updated the comment